services/wallet/bridge/hop.go

```diff
@@ -349,142 +342,50 @@ func (h *HopBridge) swapAndSend(chainID uint64, hopArgs *HopTxArgs, signerFn bin
 	return tx, err
 }
 
-// CalculateBonderFees logics come from: https://docs.hop.exchange/fee-calculation
-func (h *HopBridge) CalculateBonderFees(from, to *params.Network, token *token.Token, amountIn *big.Int, nativeTokenPrice, tokenPrice float64, gasPrice *big.Float) (*big.Int, error) {
-	amount := new(big.Float).SetInt(amountIn)
-	totalFee := big.NewFloat(0)
-	destinationTxFee, err := h.getDestinationTxFee(from, to, nativeTokenPrice, tokenPrice, gasPrice)
-	if err != nil {
-		return nil, err
-	}
+func (h *HopBridge) CalculateFees(from, to *params.Network, token *token.Token, amountIn *big.Int) (*big.Int, *big.Int, error) {
```

Comment:
They won't cause I removed these params `nativeTokenPrice, tokenPrice float64, gasPrice *big.Float` and they were used only for fees calculation for Hop, but now since we're calling hop API for that we don't need them. For other types that implement `Bridge` interface those params were already ignored.

Reply:
I meant the body of the function, not the signature

Reply:
At the end of that function it returns the token fee and bonder fee, the same as it was, we've just changed the way how we calculate them (now we're fetching them from the remote API). But all that will be undergone thorough checking.